src/style.css

```diff
@@ -1,3 +1,81 @@
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --blue-link:#00ACDC;
+}
+
 body {
   margin: 0;
 }
+
+nav {
+  display: block;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+    height: 40px;
+    width: 40px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  box-sizing: content-box;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-align: left;
+  position: relative;
+  transition: color 0.3s;
```

Comment:
You could add some line breaks / empty lines and organize the properties a bit better